Koronavilkku/Koronavilkku/Model/ExposureModel.swift

```diff
@@ -2,16 +2,18 @@ import Foundation
 import ExposureNotification
 
 struct Exposure: Codable {
-    
-    static let retentionTime: Double = (60 * 60 * 24) * 15 // 15 days
-    static let testRetentionTime: Double = 60 * 10 // 10 minutes
+    // The official exposure retention time is 10 * 24 hours from the time of exposure.
+    // As we don't know the exact time of exposure, just the 24-hour window starting
+    // from 00:00 UTC, we need to round it up to the next UTC midnight to make sure
+    // every possible exposure is covered
+    static let retentionTime: TimeInterval = (10 + 1) * 86_400
```

Comment:
For me it is more clear to read time literals that are composed from "elements" (60\*60\*24). One could define TimeInterval extensions for defining interval of days etc, but would be nice if those would be provided by the framework itself.

Reply:
Maybe like this?

Reply:
👍